stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/User.kt

```diff
@@ -47,9 +47,9 @@ public data class User(
     var role: String = "",
     var name: String = "",
     var image: String = "",
-    var invisible: Boolean = false,
```

Comment:
I would keep the default value here to avoid breaking changes on the current behavior of our customers. If anyone wants to not use this argument, they can set It up as null

Reply:
I assume the overall idea is that SDK is not supposed to do any magic like sending default values, unless those values are explicitly set by customers :)

Reply:
Yeah I think we are safe here @JcMinarro, puting the default value as nullable, should not break anything 👍 And it is more aligned to the desired behaviour